src/templates/app/user/security/account_closure.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import Loading from '../../../_common/components/loading.jsx';
+import { SeparatorLine } from '../../../_common/components/separator_line.jsx';
+import { FormRow, SubmitButton } from '../../../_common/components/forms.jsx';
+
+const AccountClosure = () => (
+    <React.Fragment>
+        <div id='msg_main' className='center-text gr-gutter gr-padding-10 invisible'>
+            <h1>{it.L('Account closure confirmed')}</h1>
+            <p className='notice-msg'>{it.L(`Accounts closed successfully. A confirmation email will be sent to your email.[_1]This page will redirect to the ${it.website_name} homepage after 10 seconds.`, '<br />')}</p>
+        </div>
+        <div id='closure-container'>
+            <div id='main-header' className='gr-padding-30'>
+                <h1 id='heading'>{it.L('Account Closure')}</h1>
+                <p>{it.L(`Closing your ${it.website_name} accounts involves closing all open positions in your accounts, and withdrawing your funds, and deactivating your accounts with ${it.website_name}`)}</p>
+            </div>
+
+            <div className='gr-no-gutter' id='closure_description'>
+                <h2 className='primary-color'>{it.L('What would you like to do?')}</h2>
+                <fieldset>
+                    <div className='gr-padding-20 gr-gutter-left gr-gutter-right'>
+                        <ClosureDescription
+                            title={it.L('Create crypto account')}
+                            subtitle={it.L('[_1]Open an account[_2] in the cryptocurrency of your choice:', `<a href="${it.url_for('new_account/realws')}">`, '</a>')}
+                            list_items={[
+                                it.L('Bitcoin (BTC)'),
+                                it.L('Ether (ETH)'),
+                                it.L('Litecoin (LTC)'),
+                                it.L('Tether (UST)'),
+                            ]}
+                        />
+                        <ClosureDescription
+                            title={it.L('Change my affiliate')}
+                            subtitle={it.L('Contact [_1]affiliates@binary.com[_2] for more info on changing your affiliate.', '<a href="mailto:affiliates@binary.com">', '</a>')}
+                        />
+                        <ClosureDescription
+                            title={it.L('Change my account limits')}
+                            subtitle={it.L('You may set limits in your account to help prevent unwanted losses.[_1]Go to [_2]self-exclusion page[_3] to manage your account limits.', '<br />', `<a href=${it.url_for('user/security/self_exclusionws')}>`, '</a>')}
+                        />
+                    </div>
+                </fieldset>
+                {/* TODO: complete these component */}
```

Comment:
Is component completed? If yes, can you please remove the comment.

Reply:
@cakasuma left these comments, but it seems completed. I'll remove it for now.

Reply:
comment still not removed?